### ISSUE TITLE
Record inference statistics

### ISSUE THREAD

**Issue by rcabanasdepaz** [PGM-Lab/InferPy#126]
Find a way to record some of the statistics of the inference. For example, after the running code, we must be able to extract and print the evolution of the loss.

```
m = pca(k, d)
opt = tf.train.AdamOptimizer(learning_rate=0.05)
inf_method = models.inference.VI(q_model(k, d), optimizer=opt, epochs=1000)
posterior = m.fit(dict(x=x_train), inf_method)
```

Other metrics might be considered.

**Comment by jcozar87** [comment]
Instead of continue with this issue, we will move to the implementation of #137
